### ISSUE TITLE
DNS Changes

### ISSUE THREAD

**Issue by mikeal** [nodeconf/collective#16]
Heya,

The DNS service we had been using since 2010 decide to go away :(

I've moved to the DNS servers on my registrar, iwantmyname, and installed their gh-pages support. However, the old sub domains will need to be added back in manually, so please let me know what you need, if anything for those.

@nodeconf/owners @nodeconf/brazil @nodeconf/budapest @nodeconf/buenos-aires @nodeconf/london @nodeconf/barcelona @nodeconf/website @thomblake

**Comment by tamzinblake** [comment]
Thanks! I assume nodeconf.com should just work if I switch it back then?

(We should eventually restructure the site tho so that nodeconf.com goes to
something collective-related)

**Comment by mikeal** [comment]
Yup, everything that is setup as gh-pages should be fine now. We’ll need to re-add all the subdomains that got pointed to things outside gh-pages tho.

**Comment by PatrickHeneise** [comment]
nodeconf/barcelona is set up as gh-page with CNAME in the repo, but doesn't seem to work.

**Comment by mikeal** [comment]
more DNS issues, let me know if this is still an issue in a few hours.

**Comment by PatrickHeneise** [comment]
barcelona is resolved, thanks!

**Comment by a0viedo** [comment]
Now nodeconf.com is working for me :confetti_ball:

**Comment by iancrowther** [comment]
London seems to be good
